src/components/cluster/detail/cluster_detail_node_pools_table.js

```diff
@@ -561,7 +574,13 @@ class ClusterDetailNodePoolsTable extends React.Component {
             </AddNodePoolWrapperDiv>
           </ReactCSSTransitionGroup>
         ) : (
-          <FlexWrapperDiv>
+          <FlexWrapperDiv className={noNodePools && 'no-nodepools'}>
+            {noNodePools && (
+              <p>
+                Add at least one node pools to this cluster so that you can
```

Comment:
I think this should say "add at least one node pool", (without the "s" at the end)

Reply:
good catch, thanks!